changelog/dep_scope_class.dd

```diff
@@ -0,0 +1,12 @@
+`scope` as a type constraint on class declarations is deprecated.
+
+`scope` as a type constraint on class declarations has been deprecated for
+quite some time.  However, starting with this release, the compiler will emit a
+deprecation warning if used.
```

Comment:
This paragraph is quite confusing without looking at the internals - it was deprecated already but now will issue deprecation warnings?

Maybe phrase it as "scheduled for deprecation"?